src/site/paragraphs/button.drupal.liquid

```diff
@@ -11,8 +11,7 @@
       }
   }
 {% endcomment %}
-<div data-template="paragraphs/button">
-  <a
-    class="usa-button-primary"
-    href="{{ entity.fieldButtonLink.url.path }}">{{ entity.fieldButtonLabel }}</a>
-</div>
+<a
+  data-template="paragraphs/button"
+  class="usa-button-primary"
+  href="{{ entity.fieldButtonLink.url.path }}">{{ entity.fieldButtonLabel }}</a>
```

Comment:
I removed the wrapper div because sometimes we will have an array of buttons that should render inline

Reply:
Do we want a `rel="noreferrer noopener"` here since the link could be external?